test/e2e/metering_manual_install_tests.go

```diff
@@ -104,6 +106,75 @@ func testManualMeteringInstall(
 	assert.NoError(t, err, "capturing logs and uninstalling metering should produce no error")
 }
 
+func DecodeMeteringConfigManifest(basePath, manifestPath, manifestFilename string) (*metering.MeteringConfig, error) {
+	manifestFullPath := filepath.Join(basePath, manifestPath, manifestFilename)
+	file, err := os.Open(manifestFullPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open the %s manifest file: %v", manifestFullPath, err)
+	}
+
+	mc := &metering.MeteringConfig{}
+	err = yaml.NewYAMLOrJSONDecoder(file, 100).Decode(&mc)
+	if err != nil {
+		return nil, err
+	}
+
+	if mc == nil {
+		return nil, fmt.Errorf("error: the decoded MeteringConfig object is nil")
+	}
+
+	return mc, nil
+}
+
+func createNFSProvisioner(ctx *deployframework.DeployerCtx) error {
+// cteate namespace
+ctx.TargetPodsCount = nonHDFSTargetPodCount
+
+_, err := ctx.Client.CoreV1().Namespaces().Get(context.Background(), ctx.Namespace, metav1.GetOptions{})
+if err != nil && !apierrors.IsNotFound(err) {
+	return nil
+}
+if apierrors.IsNotFound(err) {
+	n := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: ctx.Namespace,
+			Labels: map[string]string{
+				"name": ctx.Namespace + "-" + testingNamespaceLabel,
+			},
+		},
+	}
+	_, err = ctx.Client.CoreV1().Namespaces().Create(context.Background(), n, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	ctx.Logger.Debugf("Created the %s namespace", ctx.Namespace)
+}
+//decode and create yamls except for persistentvolume
+filepathNFSServer := "../testhelpers/nfs_server.yaml"
```

Comment:
not very fluent at all in Go, but in general we should use the absolute paths rather than relative

Reply:
Yeah agreed on using absolute paths. We expose the root directory in a test flag, so joining that path with the path to the NFS-related manifests is best.

Reply:
Yeah, it's WIP, I am not even fully sure a filepath can be valid as a map string, but I'll be making absolute paths for sure.

Reply:
Sure, that's valid:
```go
deploy.DecodeYAMLManifestToObject(filepath.join(repoDir, nfsManifestsDir, nfsServerManifestFilename), ...)
```